src/services/votes.ts

```diff
@@ -29,6 +37,13 @@ export function getVotes(dbPool: PostgresJsDatabase<typeof db>) {
   };
 }
 
+/**
+ * Retrieves the votes for a specific cycle and user.
+ * @param {PostgresJsDatabase<typeof db>} dbPool - The database connection pool.
+ * @param {string} userId - The ID of the user.
+ * @param {string} cycleId - The ID of the cycle.
+ * @returns {Promise<any[]>} A Promise resolving to an array of votes.
```

Comment:
we can remove this Promise<any[]>?  I think we can either remove this or type it better

Reply:
actually lets remove all the @returns and let typescript handle that

Reply:
Done